client/src/containers/profile/index.tsx

```diff
@@ -36,6 +34,131 @@ export const Profile = (props: RouteComponentProps) => {
 		);
 	}, [username]);
 
+	const mockPointsProps = {
+		rank: userData ? (userData.rank ? userData.rank : 9) : 9,
```

Comment:
here you can simply write:
rank: userData?.rank ?? 9

and the same for honor and completedChallenged

Reply:
nice advice, thanks